cmd/limactl/copy.go

```diff
@@ -118,24 +161,83 @@ func copyAction(cmd *cobra.Command, args []string) error {
 		for _, inst := range instances {
 			sshOpts, err = sshutil.SSHOpts("ssh", inst.Dir, *inst.Config.User.Name, false, false, false, false)
 			if err != nil {
-				return err
+				return nil, nil, err
 			}
 		}
 	} else {
 		// Copying among multiple hosts; we can't pass in host-specific options.
 		sshOpts, err = sshutil.CommonOpts("ssh", false)
 		if err != nil {
-			return err
+			return nil, nil, err
 		}
 	}
-	sshArgs := sshutil.SSHArgsFromOpts(sshOpts)
+	sshArgs = sshutil.SSHArgsFromOpts(sshOpts)
 
-	sshCmd := exec.Command(arg0, append(sshArgs, scpArgs...)...)
-	sshCmd.Stdin = cmd.InOrStdin()
-	sshCmd.Stdout = cmd.OutOrStdout()
-	sshCmd.Stderr = cmd.ErrOrStderr()
-	logrus.Debugf("executing scp (may take a long time): %+v", sshCmd.Args)
+	return sshArgs, scpArgs, nil
```

Comment:
The fact that internally we want to manage sshArgs and scpArgs does not mean we need to return 2 lists. We can make the caller life easier and the code mode consistent if we return a single list, or a `*exec.Cmd` instance created from the lists.